cypress/support/step_definitions/dashboard.cy.js

```diff
@@ -0,0 +1,35 @@
+import { When, Then } from '@badeball/cypress-cucumber-preprocessor'
+import { Dashboard } from '../po/components/Dashboard';
+
+const dashboardPage = new Dashboard();
+
+When('I add a new dashboard', () => {
+    dashboardPage.addNewDashboard();
+});
+
+When('I delete the newly added dashboard', () => {
```

Comment:
Make your steps more generic, f.e.:
'I delete the newly added dashboard' => 'I delete dashboard', 
'I edit the newly added dashboard' => 'I edit dashboard'
and so on